src/test/java/com/google/api/codegen/testsrc/frozen_dependencies.yaml

```diff
@@ -14,7 +14,7 @@ gax_version:
     lower: '0.15.0'
     upper: '0.16.0'
   nodejs:
-    lower: '0.14.0'
+    lower: '0.17.1'
```

Comment:
Oh, looks like I'm missing the correct one. But I still want to change tests as well since this change is incompatible and we'd probably want to have tests generate code that can, in theory, be executed.

Reply:
Okeydokey, up to you!